tests/xfstests/generate_report.pm

```diff
@@ -35,7 +35,7 @@ sub log_end {
 # Compress all sub directories under $dir and upload them.
 sub upload_subdirs {
     my ($dir, $timeout) = @_;
-    my $output = script_output("find $dir -maxdepth 1 -mindepth 1 -type f -or -type d");
+    my $output = script_output("if [ -d $dir ]; then find $dir -maxdepth 1 -mindepth 1 -type f -or -type d; else echo $dir folder not exist; fi");
     for my $subdir (split(/\n/, $output)) {
```

Comment:
so in case folder not exists maybe you don't need to proceed with this for loop ?

Reply:
Yes, if folder not exist I don't need to upload anything, just print a info to log to say it's not exist.

Reply:
@asmorodskyi understand where you point out. It's maybe a critical issue, I'll fix before merge.